tests/src/ViewModel/InfoBarTest.php

```diff
@@ -37,17 +50,18 @@ public function it_cannot_have_blank_text()
     /**
      * @test
      */
-    public function it_cannot_have_an_invalid_type()
+    public function dismissible_type_must_have_id()
     {
         $this->expectException(InvalidArgumentException::class);
 
-        new InfoBar('text', 'foo');
+        new InfoBar('text', InfoBar::TYPE_DISMISSIBLE, null, new DateTimeImmutable('now'));
```

Comment:
do not use dynamic date values in unit tests, this can easily be `new DateTimeImmutable('2020-01-15')`